Paintroid/src/main/java/org/catrobat/paintroid/ui/dragndrop/DragAndDropListView.kt

```diff
@@ -0,0 +1,212 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2021 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.paintroid.ui.dragndrop
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Rect
+import android.graphics.drawable.BitmapDrawable
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.View
+import android.widget.AdapterView.OnItemClickListener
+import android.widget.AdapterView.OnItemLongClickListener
+import android.widget.ListView
+import kotlin.math.max
+import kotlin.math.min
+
+private const val CONSTANT_1 = 192
```

Comment:
Hmm, since you didn't implement it, you probably won't know what this constant stands for. Me neither ^^
@ThorstenBandel, do you have any clue?

Reply:
that should be the value of the alpha channel/how transparent the layer is when its being longpressed/dragged. :)

Reply:
oh.. ok thanks :)